src/styles/product-card.scss

```
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__poster {
+    width: 160px;
+    height: 134px;
+    background-color: $secondary-color;
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-item {
+    flex-grow: 1;
+    padding-top: 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $base-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    padding-top: 4px;
+  }
+
+  &__rating {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $base-color;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 24px;
+    line-height: 18px;
+
+    &-tag {
+      font-weight: 400;
+      font-size: 12px;
+      color: $secondary-color;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      color: $base-color;
+    }
+  }
+
+  &__primary-button {
+    width: 166px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: $theme-color;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    transition: border 200ms, color 200ms, background ease-out 200ms;
+
+    &:hover {
+      border: 1px solid $theme-color;

```

you already have this border in line 86, enough just change a color.
```suggestion
      border-color: $theme-color;

```